src/Kernel/ClassOrganization.class.st

```diff
@@ -232,6 +225,15 @@ ClassOrganization >> removeProtocolIfEmpty: aProtocol [
 	self notifyOfChangedProtocolNamesFrom: oldProtocolNames to: self protocolNames
 ]
 
+{ #category : #cleanup }
+ClassOrganization >> removeUnexistingSelectorsFromProtocols [
+	"For each protocol, remove the selectors that are not present in the class."
```

Comment:
"Unexisting" in english sounds strange... but it exists: "Adjective. unexisting (not comparable) Not in existence."